src/Sarif.FunctionalTests/v2/ConverterTestData/FxCop/Readme.md

```diff
@@ -0,0 +1,15 @@
+Notes on producing test examples.
+
+This output is an FxCop run against a debug build of SARIF-SDK 
```

Comment:
>SARIF-SDK [](start = 53, length = 9)

The actual repo name is `sarif-sdk`.

Consider "... a debug build of the sarif-sdk repo."